app/views/import_batches/preview.html.erb

```diff
@@ -27,7 +27,12 @@
   <h4 class="add-bottom-margin">Running this import will</h4>
   <ul class="list-group">
     <li class="list-group-item">Create <%= pluralize(@preview.redirect_count, 'new redirect') %></li>
-    <li class="list-group-item">Create <%= pluralize(@preview.archive_count, 'new archive') %></li>
+    <li class="list-group-item">
+      Create <%= pluralize(@preview.archive_count, 'new archive') %>
+      <% if @preview.custom_archive_url_count > 0 %>
+        (<%= pluralize(@preview.custom_archive_url_count, 'with custom URL') %>)
```

Comment:
We describe this field as 'Alternative National Archives URL' in the [single edit mapping form](https://github.com/alphagov/transition/blob/4f2fe54d6753e4fcff11390c7ba8f91a75a1665e/app/views/mappings/_form.html.erb#L57) - it would be good to keep the wording consistent, although that's a longer description.